src/pages/Embark/index.tsx

```diff
@@ -297,6 +297,14 @@ export const EmbarkRoot: React.FunctionComponent<EmbarkRootProps> = (props) => {
                     }}
                     initialStore={initialStore}
                     onStoreChange={(store) => {
+                      const castedWindow = window as any
+                      if (castedWindow && castedWindow.analytics) {
+                        castedWindow.analytics.track(
+                          `Embark Store Changed - ${props.name!}`,
+                          store,
+                        )
+                      }
+
```

Comment:
i think we should do this after actually setting the data, in case this explodes for some reason